samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -72,21 +68,23 @@ private void configureSdk(CustomerIOBuilder builder, final CustomerIOSDKConfig s
             builder.cdnHost(cdnHost);
         }
 
-        if (sdkConfig.getFlushAt() != null) {
-            builder.flushAt(sdkConfig.getFlushAt());
-        }
-        if (sdkConfig.getFlushInterval() != null) {
-            builder.flushInterval(sdkConfig.getFlushInterval());
+        if (sdkConfig.isTestModeEnabled()) {
+            builder.flushInterval(1);
+            builder.flushAt(1);
+        } else {
+            if (sdkConfig.getFlushAt() != null) {
+                builder.flushAt(sdkConfig.getFlushAt());
+            }
+            if (sdkConfig.getFlushInterval() != null) {
+                builder.flushInterval(sdkConfig.getFlushInterval());
```

Comment:
Based on attached screenshots, it seems we no longer have an option to set `flushAt` and `flushInterval` anywhere in the app. If that's the case, we should remove this entirely from sample app as it's no longer needed.

Reply:
That's absolutely true, will remove them!